core/shared/src/main/scala/zio/ZIO.scala

```diff
@@ -447,8 +447,8 @@ sealed trait ZIO[-R, +E, +A] extends Serializable { self =>
   /**
    * Unwraps the optional success of this effect, but can fail with unit value.
    */
-  final def get[E1 >: E, B](implicit ev1: E1 =:= Nothing, ev2: A <:< Option[B]): ZIO[R, Unit, B] =
-    ZIO.absolve(self.mapError(ev1).map(ev2(_).toRight(())))
+  final def get[E1 >: E, B](implicit ev: A <:< Option[B]): ZIO[R, Unit, B] =
```

Comment:
Oh, I forgot this had the `E =:= Nothing` constraint. I think we should leave this method alone; we're discarding an existing error here.